src/tokens/fonts.js

```diff
@@ -1,32 +1,35 @@
 const fonts = {
   family: {
-    lato: 'Lato',
+    lato: {
+      regular: `Lato-Regular`,
```

Comment:
No backticks for static literals. Add a linting rule for this.

Reply:
We should add these rules in universe configs @kamleshchandnani